include/private/rdyconfigimpl.h

```diff
@@ -47,10 +47,17 @@ typedef enum {
   FLOW_DIFFUSION  // diffusion equation
 } RDyPhysicsFlowMode;
 
+typedef enum {
+  SOURCE_SEMI_IMPLICIT = 0,  // semi-implicit treatment
+  SOURCE_IMPLIICT_XQ2018     // implicit treatment using Xilin and Qiuhua (2018)
```

Comment:
There's a typo here and where it's used below: `SOURCE_IMPLIICT_XQ2018` should be `SOURCE_IMPLICIT_XQ2018`.